tests/__init__.py

```
@@ -0,0 +1 @@
+from pyrail.irail import irail
```

Ruff (0.8.2)

`pyrail.irail.irail` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)